lib/rules/no-inner-literal.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const findExpectCall = require('../util/find-expect-call');
+
+module.exports = function(context) {
+  return {
+    ExpressionStatement(node) {
+      let {expression} = node;
+      if (expression.type !== 'MemberExpression' && expression.type !== 'CallExpression')
+        return;
+
+      let expect = findExpectCall(expression);
+      if (!expect)
+        return;
+
+      let args = expect.arguments;
+      let [firstArgument] = args;
+      if (!firstArgument)
+        return;
+
+      let value;
+      console.log('firstArgument.type', firstArgument.type);
```

Comment:
😉

Reply:
But, but, but.... These enhance the expressiveness of the code! ;)  Fixed.